hazelcast-client-new/src/main/java/com/hazelcast/client/HazelcastClient.java

```diff
@@ -82,6 +83,9 @@ public static HazelcastInstance newHazelcastClient(ClientConfig config) {
             client.start();
             OutOfMemoryErrorDispatcher.registerClient(client);
             proxy = new HazelcastClientProxy(client);
+            if(CLIENTS.containsKey(client.getName())) {
```

Comment:
Missing whitespace in `if(`, this will fail our CheckStyle validation.

You can check this locally to safe the time waiting for the PR-Builder:
```
mvn clean compile -P findbugs
mvn clean validate -P checkstyle
```